### ISSUE TITLE
One integration should be able to be applied to multiple monitors

### ISSUE THREAD

**Issue by ghost** [a85/POSTMan-Chrome-Extension#10615]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

Several different teams at my company are using Postman with several different monitors. Each team wants all of their monitor failures reported into a specific channel for them in Microsoft Teams. For example: All of Team A's monitors should report to channel A in Microsoft Teams, all of Team B's monitors should report to channel B, etc.

Currently, there is a 1:1 relationship between monitors and integrations, forcing you to create multiple integrations to the same Microsoft Teams channel. This is a maintenance nightmare, and makes setting up a new monitor more difficult.

### Describe the solution you'd like

I would like a list of integrations to be available to choose from when setting up the monitor. To expand on the above example, there will be one integration created for Team A, one for Team B, etc. When setting up a new monitor for Team A, they can just select their integration from a dropdown, thus creating a 1:many relationship between integrations:monitors.

We are using separate workspaces for different teams, but having this feature tied to workspaces would be a big limitation. It shouldn't be tied to anything specifically so that if Team A were to have two different monitors in two different workspaces, they could still tie it to the same integration.

### Describe alternatives you've considered

I am not aware of any alternatives. I have reached out to our Postman CSM to confirm this functionality does not exist and they directed me to file a feature request here.

**Comment by abhijitkane** [comment]
Thanks @DougMurphyAbrigo - this is something we have on our roadmap. I agree with the workspace association being a limitation. In your example, do Team A and Team B use the same MS Teams "account", only with different channels to send monitor results to?

**Comment by ghost** [comment]
@abhijitkane That is correct. In my example, Team A and Team B are both teams at Abrigo and we all share the same Microsoft Teams instance. These channels will _very_ likely be in different Microsoft Teams teams, but I don't imagine that would be a factor that needs to be worried about since the integration should rely solely on the webhook URL anyway.
